Assets/Mirror/Runtime/NetworkBehaviour.cs

```diff
@@ -20,15 +20,25 @@ public class NetworkBehaviour : MonoBehaviour
         // this prevents recursion when SyncVar hook functions are called.
         bool m_SyncVarGuard;
 
+        /// <summary>True if the obejct is controlled by the client that owns it.</summary>
         public bool localPlayerAuthority { get { return netIdentity.localPlayerAuthority; } }
+        /// <summary>True if this object is running on the server, and has been spawned.</summary>
         public bool isServer { get { return netIdentity.isServer; } }
+        /// <summary>True if the object is running on a client.</summary>
         public bool isClient { get { return netIdentity.isClient; } }
+        /// <summary>True if the object is the one that represents the player on the local machine.</summary>
         public bool isLocalPlayer { get { return netIdentity.isLocalPlayer; } }
+        /// <summary>True if the obejct is only running on the server, and has been spawned.</summary>
```

Comment:
typo

Reply:
also,  this is true if the object is running in the server but it is not a client,  as opposed to running the object in host mode